Sources/VLCPlaybackController.m

```diff
@@ -504,7 +504,9 @@ - (float)playbackPosition
 
 - (void)setPlaybackPosition:(float)position
 {
-    _mediaPlayer.position = position;
+    float oldPosition = _mediaPlayer.position;
+    float diff = position - oldPosition;
+    [_mediaPlayer jumpForward:(diff * self.mediaDuration)/1000.];
```

Comment:
we should definitely fix the setting of the mediaplayer.position here instead!

Reply:
Should I remove the hack or just leave a `#warning` there?

Reply:
definitely remove it once the _mediaplayer.position works again!